presto-main/src/main/java/com/facebook/presto/operator/ThriftRpcShuffleClient.java

```diff
@@ -92,4 +94,14 @@ public void acknowledgeResultsAsync(long nextToken)
     {
         return thriftClient.abortResults(taskId, outputBufferId);
     }
+
+    @Override
+    public Throwable rewriteException(Throwable throwable)
+    {
+        // MessageTooLargeException is usually a wrapper of FrameTooLargeException; but we may test on both of them
+        if (throwable instanceof FrameTooLargeException || throwable instanceof MessageTooLargeException) {
```

Comment:
Does Drift ever throw `FrameTooLargeException`?  Looks like it's intercepted and always `MessageTooLargeException`